### PR TITLE
Np 9419/login view issue

### PR DESCRIPTION
**Pull request by microArtur**
https://nanopay.atlassian.net/browse/NP-9419

### CODE REVIEW

**Review by noodlemoodle — APPROVED** [review]

File: src/foam/u2/view/LoginView.js [hunk 1]
```diff
@@ -110,6 +110,9 @@ foam.CLASS({
   ^ .align-end {
     text-align: end;
   }
+  ^ .margin-top {
+    margin-top: 16px;
+  }
```
Comment (sarthak-marwaha) [inline src/foam/u2/view/LoginView.js]: Any problem with always having this css class enabled?

File: src/foam/u2/view/LoginView.js [hunk 2]
```diff
@@ -301,10 +304,14 @@ foam.CLASS({
         .addClass(self.myClass('content-form'))
         .callIf(self.displayWidth, function() { this.onDetach(self.displayWidth$.sub(self.resize)); })
         .startContext({ data: this }).tag(this.DATA).endContext()
-        .start()
-          .addClass('align-end')
-          .tag(this.data.SUB_FOOTER)
-        .end()
+        .callIfElse(this.data.subFooter, function() {
```
Comment (microArtur) [inline src/foam/u2/view/LoginView.js]: am i able to check the isAvailable flag somehow? So i don't need to remove the subFooter property from SignUp.js model

File: src/foam/u2/view/LoginView.js (hunk 1 above)
Comment (microArtur) [reply src/foam/u2/view/LoginView.js]: there is an extra gap 
image

File: src/foam/u2/view/LoginView.js (hunk 2 above)
Comment (jlhughes) [reply src/foam/u2/view/LoginView.js]: @sarthak-marwaha ^

File: src/foam/u2/view/LoginView.js [hunk 3]
```diff
@@ -98,6 +98,9 @@ foam.CLASS({
     width: 75%;
     padding: 2vw;
     box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
```
Comment (microArtur) [inline src/foam/u2/view/LoginView.js]: feels like 32px is too much

**Review by sarthak-marwaha — APPROVED** [review]

**Pull request merged**